SCons/Environment.py

```diff
@@ -819,24 +835,30 @@ def append_define(name, dict = dict):
             do_parse(arg)
         return dict
 
-    def MergeFlags(self, args, unique=1, dict=None):
-        """
-        Merge the dict in args into the construction variables of this
-        env, or the passed-in dict.  If args is not a dict, it is
-        converted into a dict using ParseFlags.  If unique is not set,
-        the flags are appended rather than merged.
-        """
+    def MergeFlags(self, args, unique=True):
```

Comment:
Isn't self really env?

Reply:
Yes - an instance of SubstitutionEnvironment of one of its children.